client/mm/libxc/binance.go

```diff
@@ -649,12 +658,44 @@ func (bnc *binance) getMarkets(ctx context.Context) (map[string]*bntypes.Market,
 	}
 
 	marketsMap := make(map[string]*bntypes.Market, len(exchangeInfo.Symbols))
+	tokenIDs := bnc.tokenIDs.Load().(map[string][]uint32)
 	for _, market := range exchangeInfo.Symbols {
+		dexMarkets := binanceMarketToDexMarkets(market.BaseAsset, market.QuoteAsset, tokenIDs, bnc.isUS)
+		if len(dexMarkets) == 0 {
+			continue
+		}
+		dexMkt := dexMarkets[0]
+
+		bui, _ := asset.UnitInfo(dexMkt.BaseID)
+		qui, _ := asset.UnitInfo(dexMkt.QuoteID)
+
+		var rateStepFound, lotSizeFound bool
+		for _, filter := range market.Filters {
+			if filter.Type == "PRICE_FILTER" {
+				rateStepFound = true
+				conv := float64(qui.Conventional.ConversionFactor) / float64(bui.Conventional.ConversionFactor) * calc.RateEncodingFactor
+				market.RateStep = uint64(math.Round(filter.TickSize * conv))
+				market.MinPrice = uint64(math.Round(filter.MinPrice * conv))
+				market.MaxPrice = uint64(math.Round(filter.MaxPrice * conv)) // Why was the conv outside of the math.Round?
```

Comment:
> // Why was the conv outside of the math.Round?

Not sure. Whadya expect me to test my code suggestions? 🤪

Reply:
Lol that comment was kind of meant for myself.